doc/template/darkfx/partials/affix.tmpl.partial

```diff
@@ -0,0 +1,39 @@
+{{!Copyright (c) Microsoft. All rights reserved. Licensed under the MIT license. See LICENSE file in the project root for full license information.}}
+
+<div class="hidden-sm col-md-2" role="complementary">
+  <div class="sideaffix">
+    {{^_disableContribution}}
+    <div class="contribution">
+      <ul class="nav">
+        {{#docurl}}
+        <li>
+          <a href="{{docurl}}" class="contribution-link">{{__global.improveThisDoc}}</a>
```

Comment:
It looks like this is broken (at least on the demo in your repository). The link is displayed, but doesn't point anywhere, so may want to disable it by setting `_disableContribution`?

Reply:
That will break the view source code but thanks for highlight where the improve this doc part is ^^ I'll remove it from there